stdlib/json.rb

```diff
@@ -54,7 +54,7 @@ class ParserError < JSONError
             }
 
             if (!options.parse && (klass = #{`hash`[JSON.create_id]}) != nil) {
-              klass = Opal.get(klass);
+              klass = Opal.const_get_qualified('::', klass);
```

Comment:
Is it possible to rewrite it using `const_get` from Ruby? If it's a global constant probably `Object.const_get` may  work here. For me `"::"` looks like a private api from `runtime.js` that shouldn't be used explicitly in corelib/stdlib